front/app/i18n/lv-LV.ts

```diff
@@ -0,0 +1,10 @@
+import { formatTranslationMessages } from './';
+
+const lvLVTranslationMessages = require('translations/lv-LV.json');
```

Comment:
Why isn't this file created? It should be according to https://www.notion.so/citizenlab/Translations-7e43f76d06a745b0b8455a0f7a27c9d1#4d2f83ca55004f8e81124d1864edcfc8

Reply:
I mean the `translations/lv-LV.json` file.

Reply:
Perhaps, you should trigger CI for your branch.

Reply:
Also a good idea!

Reply:
The files are already on master and so are already included in this branch (and the same for the other 2 languages), so nothing to add I guess.

Reply:
Ah indeed.